test/e2e/networks/ci.toml

```diff
@@ -80,9 +82,13 @@ privval_protocol = "tcp"
 # https://github.com/tendermint/tendermint/issues/5422
 #perturb = ["kill", "pause", "disconnect", "restart"]
 
+[node.validator06]
+misbehaviors = "double-prevote,1015"
+persistent_peers = ["validator02"]
+
```

Comment:
I think we can just use one of the existing validators, instead of adding a new one. This needs to use the new syntax for misbehaviors as well. Or if we keep it, we can just add it to the initial validator set, since it starts at the initial height.